exercises/isogram/canonical-data.json

```diff
@@ -3,7 +3,7 @@
   "comments": [
     "An isogram is a word or phrase without a repeating letter."
   ],
-  "version": "1.7.0",
+  "version": "1.7.1",
```

Comment:
I think this should be 1.8.0 as it's adding a new test case.

Reply:
You're right, I've fixed it.